packages/@react-aria/visually-hidden/src/VisuallyHidden.tsx

```diff
@@ -30,14 +30,16 @@ interface VisuallyHiddenProps extends HTMLAttributes<HTMLElement> {
 
 const styles: CSSProperties = {
   border: 0,
-  clip: 'rect(0 0 0 0)',
-  clipPath: 'inset(50%)',
-  height: 1,
-  margin: '0 -1px -1px 0',
+  clip: `rect(0 ${isAndroid() ? '10px 10px' : '0 0'} 0)`,
+  clipPath: `inset(${isAndroid() ? 40 : 50}%)`,
+  height: (isAndroid() ? 10 : 1),
+  lineHeight: 0,
```

Comment:
what was this lineHeight for?

Reply:
I did this to ensure that a button within the VisuallyHidden component will render with its hit target within the `clipPath`.